sections/projects.md

```diff
@@ -62,6 +62,14 @@ GET  /api/v1/projects/<project_id>
 GET  /api/v1/projects?sort_field=created&sort_order=ascending
 ```
 
+
+## List projects with archives
```

Comment:
⛏ I'd use "archived" so it reads "List projects with archived", also can we move this section so its below the "List projects" section.